people_in_school.rb

```diff
@@ -0,0 +1,13 @@
+class PeopleInSchool
```

Comment:
Be careful with the indentation in this class. As @salogel42 has expressed perfectly in other code reviews: 

> Don't introduce bad indentation! Indentation may seem trivial, as the program works whether you're following standard conventions or not. But when you're working on a large project with lots of other people, you want to know that everyone is following the same style guide. Even just looking back on your own code, you'll prefer well formatted code over poorly formatted code! And any workplace with code reviews in place will require that you follow their style guide, so get used to it now.